bitcoin/examples/ecdsa-psbt.rs

```diff
@@ -284,9 +286,13 @@ fn previous_output() -> TxOut {
 struct Error(Box<dyn std::error::Error>);
 
 impl<T: std::error::Error + 'static> From<T> for Error {
-    fn from(e: T) -> Self { Error(Box::new(e)) }
+    fn from(e: T) -> Self {
+        Error(Box::new(e))
+    }
```

Comment:
Please don't do formatting changes. If you didn't mean this to happen you might have your editor configured to format on save and you might be using the stable toolchain. We use a nightly toolchain to format.

Reply:
i have reversed this with the nightly toolchain